lib/rspec/core/configuration.rb

```diff
@@ -2135,6 +2135,8 @@ def load_file_handling_errors(method, file)
         suggestions = DidYouMean.new(relative_file).call
         reporter.notify_non_example_exception(ex, "An error occurred while loading #{relative_file}.#{suggestions}")
         RSpec.world.wants_to_quit = true
+      rescue SyntaxError => e
+        raise e
```

Comment:
I wasn't sure what all those other methods in the other rescues are doing. This is the functionality I need (just re-raise the error). But I don't know if there's more we want/need done to the output experience.